rex_lee.md

```diff
@@ -1,18 +1,25 @@
-## 09/07 - 09/11 ##
+## 11/09 - 11/13 ##
 
 ### This week ###
-  - Bug 1202364 - [Stingray]Cast webpage to TV (receiver side)
-  - Bug 1202365 - Ping remote webpage to TV
-  - Confirm revised UX spec of Home app
-  - code review
-  - Mentoring
+  - Bug 1221924 - [TV] FxOS TV Metrics - Treat the smart-home as an overlay instead of an app
+  - Bug 1221929 - Track the last used app before turning the device off
+  - Bug 1220567 - [TV] system Banner(Toast)
+  - Bug 1217765 - Layout for Prompting if Marketplace Content wants to access privileged APIs on the FxOS should be fix to fit TV aspect 
+  - Bug 1225062 - FTE loses focus on mulet after adjusting devlopment panel
+  - Update FTE page.2 for home app (pending for visual update)
+  - Mocked testing environment for fling website
+  - code reviewing
 
 ### Last weeks ###
-  - Build-up Fennec, apply original MWC demo patch
-  - Test and debug between Fennec and B2G for a workable Presentation API environment
-  - Read UX spec of Presentaion API scenario
-  - TV Spatial navigator article (reviewing & revising)
-  - Confirm revised UX spec of Home app
-  - code review
-  - Mentoring
-  - Bug 1202364 - [Stingray]Cast webpage to TV (receiver side)
+  - Bug 1220576 - FTE for Home app (landed)
+  - Bug 1220589 - FTE for Pin-to-home context menu button (landed)
+  - Bug 1214992 - Filter should also search cards inside folders (landed)
+  - code reviewing
+
+## 2015 Q4 Goals (Already modified based on current working
+1. presentation API bug fixing, as measured by environment establishing, reviews, and blockers burndown. 30%
+2. v2.5 features (Home, FTE, system visual, and etc), as measured by as measured by blocker burn down and days of blocker turnarounds. 30%
```

Comment:
I won't measure it by days of blocker turnarounds, because this measurement usually goes to a side-effect that people pushing issues to others. :p